mocha-client-tests/addon/tests/lib/background-messaging.test.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by hmelenok on 10/30/16.
+ */
+describe('Background', function() {
+    describe('ping', function() {
+        it('should return pong in response', function(done) {
```

Comment:
`npm run test` - runs phantomjs tests, there UI ob mocha which included as example if you install all dependencies, please read README.md before